client/webserver/api.go

```diff
@@ -292,3 +293,26 @@ func (s *WebServer) writeAPIError(w http.ResponseWriter, format string, a ...int
 	}
 	writeJSON(w, resp, s.indent)
 }
+
+// apiLogout handles the 'logout' API request.
+func (s *WebServer) apiLogout(w http.ResponseWriter, r *http.Request) {
+	err := s.core.Logout()
+	if err != nil {
+		s.writeAPIError(w, "logout error: %v", err)
+		return
+	}
+
+	http.SetCookie(w, &http.Cookie{
+		Name:    authCK,
+		Path:    "/",
+		Value:   "",
```

Comment:
I'm not sure if this matters here since you are clearing the value of the cookie and expiring it, but please `SameSite: SameSiteStrictMode`" just to be consistent and conservative.